src/tests/symlink.rs

```diff
@@ -119,6 +162,43 @@ fn test() -> io::Result<()> {
             resolver_without_symlinks.resolve(&path, request).map(|r| r.full_path());
         assert_eq!(resolved_path, Ok(path.join(request)));
     }
+}
 
-    Ok(())
+#[cfg(target_family = "windows")]
```

Comment:
```suggestion
#[cfg(target_os = "windows")]
```

Reply:
While I can do the replacement, just curious what is the convention behind this? E.g., there is an existing usage of `target_family` at Line 22:

```rust
#[cfg(target_family = "unix")]
```

Reply:
https://github.com/unrs/unrs-resolver/blob/e63a7e022bef400c1c59e99473f5072af2e6b20e/src/file_system.rs#L198

https://github.com/unrs/unrs-resolver/blob/e63a7e022bef400c1c59e99473f5072af2e6b20e/src/lib.rs#L68